exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -207,11 +207,12 @@ drill.exec: {
       // Set this property if custom absolute root should be used for remote directories, ex: root: "/app/drill".
       // root: "",
 
-      // Relative base directory for all udf directories (local and remote).
+      // Base directory for all udf directories (local and remote).
       base: ${drill.exec.zk.root}"/udf",
 
-      // Relative path to local udf directory.
-      // Generated temporary directory is used as root unless ${DRILL_TMP_DIR} or ${drill.tmp-dir} is set.
+      // Relative local udf directory. Drill temporary directory is used as root for this directory.
+      // Drill temporary directory is set to generated temporary directory
+      // unless ${DRILL_TMP_DIR} or ${drill.tmp-dir} is set.
```

Comment:
Still not sure this entirely makes sense. This is a local file system directory. We describe ow the default is based on DRILL_TMP_DIR, etc. That all make sense.

But the actual default value is based on the base directory in the file system configured by fs, which might be HDFS.

And, since we specify a value here, the `drill.tmp.dir` will never be used.

Do we really mean:
```
local: "${drill-tmp-dir}/udfs"
```